.circleci/config.yml

```diff
@@ -102,34 +144,38 @@ jobs:
               ]
             }
 
-workflows:
-  test:
-    jobs:
-      - run_pre_commit
-  terraform_pipeline:
-    jobs:
-      - terraform_lint:
-          context:
-            - slack
-      - terraform_apply:
-          requires:
-            - terraform_lint
-          context:
-            - slack
-            - gcp-dev
-      - terraform_destroy:
-          requires:
-            - terraform_apply
-          context:
-            - slack
-            - gcp-dev
-      #- git_tag:
-      #    context:
-      #      - github-repo
-      #      - slack
-      #    requires:
-      #      - terraform_destroy
-      #    filters:
-      #      branches:
-      #        only:
-      #          - master
+commands:
```

Comment:
I've been kinda anti-circleci-commands, but actually i think this use of them makes things cleaner. Very cool.